File: 1217-minimum-cost-to-move-chips-to-the-same-position/1217-minimum-cost-to-move-chips-to-the-same-position.js
/**
 * @param {number[]} position
 * @return {number}
 */
var minCostToMoveChips = function(position) {
  let odd = 0, even = 0
  position.forEach(n => n % 2 ? odd++ : even++)
  return Math.min(odd, even)
};

File: 1217-minimum-cost-to-move-chips-to-the-same-position/1217-minimum-cost-to-move-chips-to-the-same-position.go
func minCostToMoveChips(position []int) int {
  odd, even := 0, 0
  for _, v := range position {
    if v % 2 == 1 {
      odd++
    } else {
      even++
    }
  }
  if odd > even {
    return even
  } else {
    return odd
  }
}